### ISSUE TITLE
Improvement - needs discussion - buttons for adding or removing items in the cart can be improved

### ISSUE THREAD

**Issue by DagA11y** [it-vegard/a11y-reference-website#109]
Imagine having multiple items in cart and using a screen-reader. Navigating by buttons alone will produce a list of buttons;

- button +
- button -
- button +
- button -
- button +
- button -
- button +
- button -

I think we should add some context to the buttons, otherwise this is not exactly usable for screen-reader users.

aria-label, aria-describedby seems like a good choice, but needs to be discussed as they can be problematic for voice-navigation.

I liked the article from Adrian Roselli on the subject of accessible cart tables;

https://adrianroselli.com/2022/01/accessible-cart-tables.html

**Comment by it-vegard** [comment]
Yes, I fully agree. We also need to add feedback to screen reader users when adding items to the cart, or updating (adding/removing items). Could it work by changing the button text to "add" and "remove", and using aria-describedby to postfix the product name? So we have "Add <product name>" and "Remove <product name>"?

Pull requests are appreciated, but the checkout process is also my first priority when I start working on the project again next week. (I am currently occupied with a conference). Everything from adding to cart and through "purchase" was kind of hasty work as money dried up in the original project... 😅I'd like to fix that now.
